eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -9,6 +9,13 @@
          This version does not have source build intermediate.
          When a newer version is picked up, the intermediate should be utilized. -->
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.AnalyzerUtilities/*3.3.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Build/17.7.2" />
```

Comment:
Not sure if these should not also be added to the `Versions.Details.xml` to allow the product source-build to use the previously built versions. Maybe @mthalman would know better.

Reply:
Pointed out in https://github.com/dotnet/format/pull/2122#discussion_r1502974053